mk/main.mk

```diff
@@ -170,7 +170,10 @@ RUST_LIB_FLAGS_ST3 += -C prefer-dynamic
 
 # Landing pads require a lot of codegen. We can get through bootstrapping faster
 # by not emitting them.
-RUSTFLAGS_STAGE0 += -Z no-landing-pads
+
+ifdef CFG_DISABLE_STAGE1_LANDING_PADS
```

Comment:
Could this be called stage0 to mirror the option it's used for below? I guess it kinda depends on what a "stage1 bootstrap" means, which also depends on who you talk to! May be worth just keeping these numbers in sync though to alleviate confusion of the off-by-one nature?